.rubocop_todo.yml

```diff
@@ -1,34 +1,47 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2015-10-20 14:00:00 -0700 using RuboCop version 0.34.2.
+# on 2015-10-27 16:19:03 -0700 using RuboCop version 0.34.2.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
-require: rubocop-rspec
 
-# Offense count: 1
+# Offense count: 4
```

Comment:
I wonder if we can simplify or break out the `add_folder` / `add_box` logic, so we're not creating more things to refactor later. Care to work on this together tomorrow?